ee/clickhouse/views/experiment_holdouts.py

```diff
@@ -50,6 +50,20 @@ def _get_filters_with_holdout_id(self, id: int, filters: list) -> list:
             )
         return updated_filters
 
+    def validate_filters(self, filters):
+        for filter in filters:
+            if "rollout_percentage" in filter:
+                rollout_percentage = filter.get("rollout_percentage")
+                if rollout_percentage is None:
+                    raise serializers.ValidationError("Rollout percentage must be present.")
+                if rollout_percentage < 0 or rollout_percentage > 100:
+                    raise serializers.ValidationError("Rollout percentage must be between 0 and 100.")
+
+            else:
+                raise serializers.ValidationError("Rollout percentage must be present.")
```

Comment:
`filter.get("rollout_percentage")` returns `None` if the key `rollout_percentage` does't exist, so the check `if "rollout_percentage" in filter:` is redundant.

```suggestion
            rollout_percentage = filter.get("rollout_percentage")
            if rollout_percentage is None:
                raise serializers.ValidationError("Rollout percentage must be present.")
            if not (0 <= rollout_percentage <= 100):
                raise serializers.ValidationError("Rollout percentage must be between 0 and 100.")
```